src/mudlet.cpp

```diff
@@ -1913,7 +1915,6 @@ void mudlet::show_options_dialog()
     connect(dactionReconnect, SIGNAL(triggered()), pDlg->need_reconnect_for_data_protocol, SLOT(hide()));
     connect(actionReconnect, SIGNAL(triggered()), pDlg->need_reconnect_for_specialoption, SLOT(hide()));
     connect(dactionReconnect, SIGNAL(triggered()), pDlg->need_reconnect_for_specialoption, SLOT(hide()));
-    if( ! pDlg ) return;
```

Comment:
Oh, so it doesn't! :open_mouth: _Unless_ we (use `#include <new>`?) and/or tag the `dlgProfilePreference` contructor with `Q_DECL_NOEXCEPT`?

Something _new_ for me to learn today. :smile: 

Anyhow we will need to return to this area to fix that issue I mentioned and a member pointer that is set to `nullptr` when there is not an instance around does seem the simplest way to ensure only one `dlgProfilePreference` instance is ever present at a time.

That does not distract however from the validity of this change. :+1: